sig-robotics/README.md

```diff
@@ -7,11 +7,29 @@ The [charter](charter.md) defines the scope and governance of the Robotics Speci
 
 ## Meetings
 
-
-* Regular SIG Meeting: [Tuesday at 10:00 UTC+8](https://zoom.us/my/kubeedge) (weekly, starts on 22th, June 2021). [Convert to your timezone](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=GMT%2B8&).
- * [Meeting notes and Agenda]().
- * [Meeting Calendar]() | [Subscribe]()
+* Regular SIG Meeting: [Tuesday at 10:00 UTC+8](https://zoom.us/my/kubeedge) (biweekly, starts on 22th, June 2021). [Convert to your timezone](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=GMT%2B8&).
+ * [Meeting notes and Agenda](https://docs.google.com/document/d/1i-DTZTsj2GjKHlocpkyzfEO--zYvx7sZ4bEFoJCy4YM/edit#).
+ * [Meeting Calendar](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MGRnMGJzdjhmY3E2aDhhOTZlOXV0M211dXBfMjAyMzAyMDdUMDIwMDAwWiBodWFuZzY4OTRAbQ)
 
 ## Contact
-- Slack: [#sig-robotics]()
+
+- [#sig-Robotics slack channel](https://join.slack.com/t/kubeedge/shared_invite/enQtNjc0MTg2NTg2MTk0LWJmOTBmOGRkZWNhMTVkNGU1ZjkwNDY4MTY4YTAwNDAyMjRkMjdlMjIzYmMxODY1NGZjYzc4MWM5YmIxZjU1ZDI)
+- [Open Community Issues](https://github.com/kubeedge/community/issues)
 - [Mailing list](https://groups.google.com/forum/#!forum/kubeedge)
+
+## Leadership
+
+### Chairs
+
+The Chairs of the SIG run operations and processes governing the SIG.
+
+*TODO*
+
+### Technical Leads
+
+The Technical Leads of the SIG establish new subprojects, decommission existing subprojects, and resolve cross-subproject technical issues and decisions.
+
+- Zhengguang Ou (@ugvddm), Huawei
+- JoeyHwong (@JoeyHwong-gk)
+
+> The co-chair is still open for recruitment. If any individual or organization is willing to participate in our community work, please send an email to [kubeedge](kubeedge@googlegroups.com).
```

Comment:
I am not sure how much I can commit this, but would like to join if I am eligible to this participation.

Reply:
Awesome, thanks for reaching out! We're absolutely thrilled that you're interested in joining our community, and we're excited to see the unique perspectives and contributions you'll bring to the table.
We think your background is a perfect fit for our community, and we can't wait to have you on board. 

Regarding the `co-chair` selection, we'll reach out to the KubeEdge TSC (@kevin-wangzefeng @fisherxu) to create a document outlining the guidelines and requirements for the role, and we'll make sure to publish it in the community to keep everyone in the loop.

Thanks again for your enthusiasm and interest in KubeEdge - we're lucky to have you!